packages/data/config/navi-config.d.ts

```diff
@@ -1,12 +1,16 @@
 import { Grain } from 'navi-data/utils/date';
 declare module 'navi-config' {
-  export type BaseDataSource<Type, Options = void> = {
-    type: Type;
+  export type DataSourceNamespace = {
     name: string;
     displayName: string;
     description?: string;
+  };
+
+  export type BaseDataSource<Type, Options = void> = DataSourceNamespace & {
+    type: Type;
     uri: string;
     options?: Options;
+    namespaces?: DataSourceNamespace[];
     suggestedDataTables?: string[];
```

Comment:
Maybe move `suggestedDataTables` to `DataSourceNamespace`?

Reply:
```suggestion
```

Reply:
That's what I did initially, I just bumped it to the root level because in the future we're not going to have `namespaces` at all and we'll have to somehow define all suggested tables at the root level
But yeah maybe for now it makes more sense to have it in `DataSourceNamespace`